src/core/core.helpers.js

```diff
@@ -173,17 +149,16 @@ module.exports = function(Chart) {
 	helpers.getValueOrDefault = function(value, defaultValue) {
 		return value === undefined ? defaultValue : value;
 	};
-	helpers.indexOf = function(arrayToSearch, item) {
-		if (Array.prototype.indexOf) {
-			return arrayToSearch.indexOf(item);
-		} else {
-			for (var i = 0; i < arrayToSearch.length; i++) {
-				if (arrayToSearch[i] === item)
+	helpers.indexOf = Array.prototype.indexOf?
```

Comment:
There are a few direct calls of `indexOf` on arrays in the code, should we change these calls to use `helpers.indexOf` or should we remove this polyfill?